lib/cosmos/win32/excel.rb

```diff
@@ -53,14 +84,18 @@ def initialize (filename)
       GC.start
     end
 
-  end
-
-  module ExcelColumnConstants
```

Comment:
Removing this will definitely break existing code

Reply:
I'll add this back.